WordPress/AbstractArrayAssignmentRestrictionsSniff.php

```diff
@@ -175,7 +176,9 @@ public function process_token( $stackPtr ) {
 				$inst[ $key ][] = array( $val, $token['line'] );
 			}
 		} elseif ( \in_array( $token['code'], array( \T_CONSTANT_ENCAPSED_STRING, \T_DOUBLE_QUOTED_STRING ), true ) ) {
-			// Covers assignments via query parameters: `$foo = 'bar=taz&other=thing';`.
+			/*
+			 * Covers assignments via query parameters: `$foo = 'bar=taz&other=thing';`.
+			 */
```

Comment:
Why switch to multi-line comment format for this single line comment?

Reply:
Because the `if` branch of the control structure also has a multi-line comment. This makes it clearer that both "branches" are handling different cases, instead of both handling the same case in different ways.